build.sbt

```diff
@@ -128,6 +128,7 @@ lazy val commonSettings = clearSourceAndResourceDirectories ++ publishSettings +
   // we always assume that Java classes are standalone and do not have any dependency
   // on Scala classes
   compileOrder := CompileOrder.JavaThenScala,
+  incOptions := incOptions.value.withNameHashing(false).withAntStyle(true), // until we have class-based name hashing
```

Comment:
you could put it in a different file (`local.sbt`) and add that filename to your global `~/.gitignore`. also for me, it only worked with `in ThisBuild`, so the full line would be

```
(incOptions in ThisBuild) := (incOptions in ThisBuild).value.withNameHashing(false).withAntStyle(true)
```